crypto/src/main/java/org/web3j/crypto/BlobUtils.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2024 Web3 Labs Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package org.web3j.crypto;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.UncheckedIOException;
+import java.nio.ByteBuffer;
+import java.nio.charset.StandardCharsets;
+
+import ethereum.ckzg4844.CKZG4844JNI;
+import org.apache.tuweni.bytes.Bytes;
+
+public class BlobUtils {
+
+    static {
+        CKZG4844JNI.loadNativeLibrary();
+    }
+
+    private static final byte blobCommitmentVersionKZG = 0x01;
+
+    public static void loadTrustedSetupParameters() {
```

Comment:
If we call this once, do we ever need to call it again?

Could we not just call this after CKZG4844JNI.loadNativeLibrary() above, and never call freeTrustedSetup() ?

Reply:
Yes sounds like a better approach

Reply:
Given both option, BlobUtil will automatically load file in static method. If someone wants to use his own file, he can free the setup once and give path to the file